### PR TITLE
[fix] #2013: Hotfix CLI args.

### LINKED ISSUE #2013: Fix empty CLI args not recognised
Sometimes an empty string is being passed to commands as part of `exec`.

### PR DESCRIPTION
**Pull request by appetrosyan**
Signed-off-by: Aleksandr <a-p-petrosyan@yandex.ru>

### Description of the Change

- Accept empty args which are sometimes passed by `exec`. 
- Fixed CI coverage workflow. 
- Generate changelog for the March Release

### Issue

Closes #2013

### Benefits

Changelog, working CLI args, working push workflow. 

### Possible Drawbacks

None

### CODE REVIEW

**Review by s8sato — DISMISSED** [review]

**Review by s8sato — APPROVED** [review]

File: CHANGELOG.rst
Comment (Arjentix) [inline CHANGELOG.rst:14]: Why does @s8sato is mentioned by it's Github name and others by their real names? Also sometimes you use full name and someties just first name. I think we need one style. At least for codeowners

File: CHANGELOG.rst
Comment (Arjentix) [inline CHANGELOG.rst:11]: Now there are also by-call triggers

**Review by Arjentix — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst
Comment (appetrosyan) [reply CHANGELOG.rst]: > Why does @s8sato is mentioned by it's Github name and others by their real names?

Because of how he signs-off his commits. 

```
Signed-off-by: s8sato <49983831+s8sato@users.noreply.github.com>
```

> Also sometimes you use full name and someties just first name

This is because some of us (myself included) have used 
```
git config --global user.name Aleksandr Petrosyan
```
when we were supposed to do 

```
git config --global user.name "Aleksandr Petrosyan"
```

I don't mind it having only my first name, but if you want to always have your full name, you can amend the signoff on your commits.

**Review by s8sato — APPROVED** [review]

**Review by Arjentix — APPROVED** [review]

**Pull request merged**
